package.json

```diff
@@ -3,7 +3,6 @@
     "@babel/core": "^7.0.0",
     "@babel/node": "^7.0.0",
     "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
```

Comment:
This package is no longer referenced in the main workspace project and is now added as a devDependency of `babel-preset-gatsby`.